scheduler/workflow.go

```diff
@@ -431,7 +452,12 @@ func submitPublishJob(pj job, t *task, wg *sync.WaitGroup, pu *publishNode) {
 	// Decrement the waitgroup
 	defer wg.Done()
 	// Create a new process job
-	j := newPublishJob(pj, pu.Name(), pu.Version(), pu.InboundContentType, pu.config.Table(), t.metricsManager, t.id)
+	var j job
+	if pu.Hostname != "" {
+		j = newPublishJob(pj, pu.Name(), pu.Version(), pu.InboundContentType, pu.config.Table(), t.RemoteManagers.Get(pu.Hostname), t.id)
```

Comment:
Do we need to treat these as separate? We are talking RPC for local instances, so if a task manifest does not specify a hostname or target for the workflow, then we should default it to to the local instance and just make the same call regardless if it's a remote instance or a local instance.